daemon/logger/fluentd/fluentd.go

```diff
@@ -0,0 +1,40 @@
+package fluentd
+
+import (
+	"github.com/docker/docker/daemon/logger"
+	"github.com/fluent/fluent-logger-golang/fluent"
+)
+
+type Fluentd struct {
+	containerid string
+	writer      *fluent.Fluent
+}
+
+func New(tag string) (logger.Logger, error) {
+	log, err := fluent.New(fluent.Config{FluentPort: 24224, FluentHost: "localhost"})
+	if err != nil {
+		return nil, err
+	}
+	return &Fluentd{
+		containerid: tag,
+		writer:      log,
+	}, nil
+}
+
+func (f *Fluentd) Log(msg *logger.Message) error {
+	tag := "docker." + f.containerid + "." + msg.Source
+	var data = map[string]string{
+		"source": msg.Source,
+		"log":    string(msg.Line),
+	}
+	f.writer.PostWithTime(tag, msg.Timestamp, data)
```

Comment:
sorry, should have just looked at the package... looks like it does some buffering and reconnecting... still think it should be documented (in the docker docs) though.